src/main/java/io/github/lambdatest/gradle/AppUploader.java

```diff
@@ -19,35 +19,46 @@ public AppUploader(String username, String accessKey, String appFilePath) {
     }
 
     public CompletableFuture<String> uploadAppAsync() {
-        return CompletableFuture.supplyAsync(() -> {
-            try {
-                OkHttpClient client = new OkHttpClient().newBuilder().build();
-                MediaType mediaType = MediaType.parse("application/octet-stream");
-                RequestBody body = new MultipartBody.Builder().setType(MultipartBody.FORM)
-                        .addFormDataPart("appFile", appFilePath,
-                                RequestBody.create(mediaType, new java.io.File(appFilePath)))
-                        .addFormDataPart("type", "espresso-android")
-                        .build();
-                Request request = new Request.Builder()
-                        .url(Constants.API_URL)
-                        .method("POST", body)
-                        .addHeader("Authorization", Credentials.basic(username, accessKey))
-                        .build();
-                try (Response response = client.newCall(request).execute()) {
-                    if (!response.isSuccessful()) throw new IOException("Unexpected code " + response);
+        return CompletableFuture.supplyAsync(
+                () -> {
+                    try {
+                        OkHttpClient client = new OkHttpClient().newBuilder().build();
+                        MediaType mediaType = MediaType.parse("application/octet-stream");
+                        RequestBody body =
+                                new MultipartBody.Builder()
+                                        .setType(MultipartBody.FORM)
+                                        .addFormDataPart(
+                                                "appFile",
+                                                appFilePath,
+                                                RequestBody.create(
+                                                        mediaType, new java.io.File(appFilePath)))
+                                        .addFormDataPart("type", "espresso-android")
+                                        .build();
+                        Request request =
+                                new Request.Builder()
+                                        .url(Constants.API_URL)
+                                        .method("POST", body)
+                                        .addHeader(
+                                                "Authorization",
+                                                Credentials.basic(username, accessKey))
+                                        .build();
+                        try (Response response = client.newCall(request).execute()) {
+                            if (!response.isSuccessful())
```

Comment:
Just checking,
can we add a check if `response is successful and responseBody is null` 
@Delta456 , any thoughts on this ?

Reply:
Ah my bad, this is just the format changes!
Ignore above!